src/color/border.json

```diff
@@ -66,20 +66,27 @@
         }
       },
       "Focus": {
-        "onLight": {
+        "grey": {
```

Comment:
I am having a hard time swallowing 'grey' in this context. Mainly this descriptive of color in a series of tokens that is mainly use case driven. Also, this is not really "gray" as this is black with an opacity. 

This was previously onLight, so what is the new use case for this border? The token is auro-color-border-focus-gray? This all feels anti-pattern.